JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -228,8 +228,8 @@ public function denormalize($data, $class, $format = null, array $context = [])
         );
 
         foreach ($normalizedData as $attributeName => $attributeValue) {
-            // Ignore JSON-LD special attributes
-            if ('@' === $attributeName[0]) {
+            // Ignore JSON-LD special attributes and null attribute names
+            if ('@' === $attributeName[0] || is_null($attributeName[0])) {
```

Comment:
No, it's not relevant to me anymore.  Also, I cannot write a test because I
just have no idea.

The issue was related to a different application and I think there was a
property exposed that should not have been. That has been fixed in the
application. I created this issue because initially we were not sure the
nature of the issue.
On Mar 11, 2016 2:37 AM, "Samuel ROZE" <notifications@github.com> wrote:

> In JsonLd/Serializer/ItemNormalizer.php
> <https://github.com/api-platform/core/pull/394#discussion_r55814401>:
>
> > @@ -228,8 +228,8 @@ public function denormalize($data, $class, $format = null, array $context = [])
> >          );
> >
> >          foreach ($normalizedData as $attributeName => $attributeValue) {
> > -            // Ignore JSON-LD special attributes
> > -            if ('@' === $attributeName[0]) {
> > +            // Ignore JSON-LD special attributes and null attribute names
> > +            if ('@' === $attributeName[0] || is_null($attributeName[0])) {
>
> @dylanjacob <https://github.com/dylanjacob> is it still relevant to you?
> Like @dunglas <https://github.com/dunglas>, I would like to see why this
> check is required... Moreover, can you please write a test?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/api-platform/core/pull/394/files#r55814401>.
>